webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/index.tsx

```diff
@@ -4,31 +4,64 @@
 import classNames from 'classnames';
 import React, {useCallback, useMemo} from 'react';
 import {FormattedMessage, useIntl} from 'react-intl';
+import {useDispatch} from 'react-redux';
 
 import ChevronDownIcon from '@mattermost/compass-icons/components/chevron-down';
 import type {SchedulingInfo} from '@mattermost/types/schedule_post';
 
+import {closeModal, openModal} from 'actions/views/modals';
+
 import * as Menu from 'components/menu';
 import Timestamp from 'components/timestamp';
 
+import {ModalIdentifiers} from 'utils/constants';
+
 import './style.scss';
+import ScheduledPostCustomTimeModal from '../scheduled_post_custom_time_modal/scheduled_post_custom_time_modal';
 
 type Props = {
     disabled?: boolean;
-    onSelect: (e: React.FormEvent, schedulingInfo: SchedulingInfo) => void;
+    onSelect: (schedulingInfo: SchedulingInfo) => void;
 }
 
 export function SendPostOptions({disabled, onSelect}: Props) {
     const {formatMessage} = useIntl();
+    const dispatch = useDispatch();
 
     const handleOnSelect = useCallback((e: React.FormEvent, scheduledAt: number) => {
+        e.preventDefault();
+        e.stopPropagation();
```

Comment:
Honestly, I don't think there's any reason for it. Just out of habit. We know that we don't want to do anything with this event anywhere else. Any idea unnecessary stopPropagation could cause any issue?

Reply:
Not for form events at least since I just checked and you can't even nest them. It could matter for other types like a mouse or keyboard event, but that's not what we have here